packages/canary-web-components/src/components/ic-pagination-bar/ic-pagination-bar.tsx

```diff
@@ -500,7 +499,7 @@ export class PaginationBar {
                   onInput={() => this.handleInputChange()}
                   max={this.totalPages}
                   min="1"
-                  validationInlineInternal
```

Comment:
i think the prop name is still `validationInlineInternal`

Reply:
Added back.